ext/bigdecimal/bigdecimal.c

```diff
@@ -3500,6 +3500,9 @@ rb_convert_to_BigDecimal(VALUE val, size_t digs, int raise_exception)
  *  in the value, the result is rounded to that number of digits,
  *  according to the current rounding mode; see BigDecimal.mode.
  *
+ *  When +ndigits+ is 0, number of digits to correctly represent a float number
```

Comment:
```suggestion
 *  When +ndigits+ is 0, the number of digits to correctly represent a float number
```

Adding this "the" makes this harmonize with the other file's change in wording.

Reply:
Thanks!